src/components/Dropdown/Dropdown.scss

```diff
@@ -59,16 +58,16 @@ $border-width: 1px;
     }
 
     &.moonstone-opened {
-        box-shadow: inset 0 0 0 $border-width var(--color-accent);
+        border: var(--selector-border);
```

Comment:
You should use `--selector-border_active` instead of `--selector-border` as we want to highlight the dropdown when the component displays its menu.